### DISCUSSION TITLE
FED Calculation / FLD

### DISCUSSION THREAD

**Discussion by ShehabGamrah** [firemodels/fds#12911] [Q&A]
Hey everyone, just have a question regarding the calculation of FED. 
In  the FDS Guide this is the calculation: FEDtot = (FEDCO +FEDCN +FEDNOx +FLDirr)×HVCO2 +FEDO2
I am modelling a polyurethane fire with the following reaction formula: C6.3H7.1O2.1N1.0 and byproducts: CO, SOOT, and HCN. If I included FED devices does this include into account FEDNOx. Also if I want to consider the influence of HCL, how to incorporate. Lastly, is FLD computed within the equation or do I have to specify it.

**Comment by drjfloyd** [comment]
If you do not specify a REAC that produces NO or NO2, then there will be none made to use in the equation. The simple chemistry model does not produce NOx.

If you want to use a modified FED equation you can:
-Output the gasses of interest at the locations of interest and do your equation in post processing
-Use CTRL and DEVC functions to calculate your modified equation
-Modify the FDS source code and compiled your own version of FDS

**Reply by ShehabGamrah** [reply]
Thanks. Using the simple chemistry model, will HCL be computed from the FLD

**Reply by drjfloyd** [reply]
The FLD uses HCL. As with NOx you would have to have a REAC that made HCL, and simple chemistry does not make HCL.   If you have some FED equation you wish to use that is not either the FED or FLD equation in FDS where it has terms for both HCL and NOx in the same equation, then you will need to do one of the three things I previously stated.
